test/entrypoint/srv_select/storage_init.lua

```diff
@@ -227,4 +228,100 @@ return function()
         unique = false,
         if_not_exists = true,
     })
+
+    if datetime_supported then
+        local datetime_format = {
+            {name = 'id', type = 'unsigned'},
+            {name = 'bucket_id', type = 'unsigned'},
+            {name = 'datetime_field', type = 'datetime'},
+        }
+
+
```

Comment:
It was intentional since there is a hundred lines of code and it would be nice to visually separate one space setup from the other. But I can remove them, if you insist.